### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by RainRat**

### CODE REVIEW

File: tests/fixtures.py [hunk 1]
```diff
@@ -775,7 +775,7 @@ def parametrize(params, values, **kwargs):
         ("kernel32", "file", capa.features.file.Export("lstrlenW"), True),
         ("kernel32", "file", capa.features.file.Export("nope"), False),
         # forwarded export
-        ("ea2876", "file", capa.features.file.Export("vresion.GetFileVersionInfoA"), True),
```
Comment (williballenthin) [inline tests/fixtures.py:778]: this is not a typo, please revert this one line

**Review by williballenthin — CHANGES_REQUESTED** [review]
tons of good fixes here, thank you!

please revert the one change indicated, then we'll be happy to merge!

File: tests/fixtures.py (hunk 1 above)
Comment (mr-tz) [reply tests/fixtures.py:778]: @RainRat bump

**Comment by mr-tz** [comment]
Thank you, have you signed the CLA?

> 📝 If you are not currently covered under a CLA, please visit https://cla.developers.google.com/. Once you've signed, follow the "New Contributors" link at the bottom of that page to update this check.

**Comment by mr-tz** [comment]
Hey @RainRat, are you able to sign the CLA? Otherwise, we have to close this PR, unfortunately.

**Comment by RainRat** [comment]
I have signed it, but since my Google email is different from my Github email, it does not recognize.

I believe that the *changes* are too small to be copyrightable, and if they are large enough, my contribution is released to the public domain. Feel free to take them and make a new commit with them if you want.

**Comment by RainRat** [comment]
I messed it up and will have to start over

**Comment by mr-tz** [comment]
ok, thank you!

**Pull request closed without merging**